README.md

```diff
@@ -5,23 +5,251 @@
+Similarly, the operator API uses the same profiles, which can be selected as a starting point for the installation.   
+For comparison, here is an installation of the SDS profile using helm:
+
+```bash
+helm template install/kubernetes/helm/istio --name istio --namespace istio-system \
+    --values install/kubernetes/helm/istio/values-istio-sds-auth.yaml | kubectl apply -f -
+```
+
+The same installation would be expressed as a CustomResource (CR) through the new API as:
+
+```yaml
+# sds-install-cr.yaml
+
+apiVersion: install.istio.io/v1alpha1
+kind: IstioControlPlane
+metadata:
+  name: istio-operator-config
+  namespace: istio-system
+spec:
+  profile: sds
+```
+
+The profile can be installed into the cluster by installing the operator into a pod into the cluster and applying the above CR, or applying it via the CLI mode. When using CLI mode, it's not necessary to specify the entire CR, only the spec portion i.e.
+
+```yaml
+# sds-install.yaml
+
+profile: sds
+```
+
+In the absence of a profile, the default demo profile is installed. All profiles are available compiled in, or the profile can point to a local file path.
+
+## Developer quick start
+
+#### Installation 
+
+The latest, not fully reviewed code is in github.com/ostromart/istio-installer. PRs into the official repo at istio.io/operator are lagging by a few days. 
+
+```bash
+git clone https://istio.io/operator.git
+cd istio-installer
+go build -o <your bin path> ./cmd/iop.go
+```
+
+#### Flags
+
+The iop command supports the following flags:
+
++   logtostderr: logs to console (by default logs go to ./iop.go)
++   dry-run (default is true for now): console output only, nothing applied to cluster or written to files
++   verbose: display entire manifest contents and other debug info
+
+#### Basic default manifest
+
+```bash
+iop manifest
+```
+
+This generates a manifest with all compiled in defaults (charts and base profile). You can see these sources in the repo under data/[profiles|charts]. 
+
+#### Output to dirs
+
+The above output is concatenated into a single file. To generate a directory hierarchy, with subdirectory levels representing a child dependency, use:
+
+```bash
+iop manifest -o istio_manifests
+```
+
+This will create a dir hierarchy which should be traversed DFS when applying the yamls. Child manifest directories must wait for the parent, but not sibling manifest directories.
+
+#### Just apply it for me
+
+```bash
+iop install
+```
+
+This command will generate the manifests and apply them in the correct dependency order, waiting for dependencies to have CRDs available.   
+Note: right now the actual "kubectl apply" if only displayed but not run by default. Set --dry-run=false to actually apply to cluster. 
```

Comment:
Please rewrite the note because it is not clear what it is trying to convey.

Reply:
Done.